### ISSUE TITLE
可否就Two_stage_TrAdaboost_R2结果作简要说明

### ISSUE THREAD

**Issue by qinhaohcen** [Bin-Cao/TrAdaboost#1]
可否就Two_stage_TrAdaboost_R2结果作简要说明，谢谢

**Comment by Bin-Cao** [comment]
如果您没有其他问题，我将关闭此issue

**Comment by qinhaohcen** [comment]
举一个例子，比如在exp6.py这个文件中，计算所得出的MSE为82.98，这个数据应该处在什么范围内证明算法的的有效性？

**Comment by Bin-Cao** [comment]
MSE : Mean Square Error 
MSE = 1/n * (sum( (yi - pre_yi) ^ 2 ) for i in range(1-n)
其中 yi 为真实响应， pre_yi为预测响应
可以看出MSE值与真实真实响应y的量级有直接关系。
例如 :
y的大小在1000左右， MSE 为10说明预测精度很好。若y的大小在10左右， MSE 为10说明预测很差。

代码中采用MSE比较不同模型的回归效果，哪个相对好。
迁移学习后的回归结果是否精准， 预测有效性多大，可以使用多个回归评价指标来判断。一般我们习惯采用决定系数 R2，MAPE, MSE等，sklearn 官网提供丰富的评价指标计算端口 ： scikit-learn.org/stable/search.html?q=metric

不同的指标各有优缺点，可以参考材料信息学导论(I) （线性回归）一章。
